E2E/pageObjects/login.page.ts

```diff
@@ -54,4 +55,28 @@ export = {
     login.loginIntoCreatedGameRoom(url, username);
     roles.spectatorLogin();
   },
+
+  voterUploadAvatarDuringLogin: (username: string, avatarImage: string) => {
+    I.amOnPage('/');
+    I.see('Welcome! Let’s begin.');
+    I.waitForVisible(locator.inputs.username);
+    I.fillField(locator.inputs.username, username);
+    I.fillField(locator.inputs.avatar, avatarImage);
+    I.click(locator.buttons.login);
+    I.waitForText('Choose your role');
+    roles.voterLogin();
+    I.waitForText('Waiting for more players');
+  },
+
+  spectatorUploadAvatarDuringLogin: (username: string, avatarImage: string) => {
+    I.amOnPage('/');
+    I.see('Welcome! Let’s begin.');
```

Comment:
Same text is used multiple times. It would be useful to add it as separate parameter